changes.txt

```diff
@@ -1,5 +1,22 @@
 CHANGES
 
+2015-04-09
+- Added Elastica\Query\SimpleQueryString::setMinimumShouldMatch
+- Added Elastica\Query\FunctionScore::setMinScore
+- Added Elastica\Query\MoreLikeThis::setMinimumShouldMatch
+
+2015-04-08
+- Added new methods to Elastica\Aggregation\DateHistogram: setOffset, setTimezone
+- Following methods in Elastica\Aggregation\DateHistogram marked as deprecated: setPreOffset, setPostOffset, setPreZone, setPostZone, setPreZoneAdjustLargeInterval
+
+2015-04-07
+- [BC break] Elastica\Query\QueryString::setLowercaseExpandedTerms removed
```

Comment:
Do you know in which version of ES this was removed? I didn't check the changelog ...